security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -267,49 +267,49 @@ Continuous Integration (CI) servers should be isolated and restricted to project
 
 Supply chain tools can gather and sign build pipeline metadata. Later stages can then verify the signatures to validate that the prerequisite pipeline stages have run.
 
-The reader should ensure that the CI and Continuous Delivery (CD) infrastructure is as secure as possible. For example, security updates should be prioritized to be installed, and cryptographic keys should be protected from exfiltration via the use of Hardware Security Modules (HSM) or Credential Managers.
+The DevOps team should ensure that the CI and Continuous Delivery (CD) infrastructure is as secure as possible. For example, security updates should be prioritized to be installed, and cryptographic keys should be protected from exfiltration via the use of Hardware Security Modules (HSM) or Credential Managers.
```

Comment:
A DevOps team is a very specific set of individuals, where as the reader (audience) of the whitepaper is much more expansive than a DevOps team.  The reader is assuming some level of responsibility, either directly in their role, or indirectly as a result of reading the paper (delegating the work, knowing it needs done, etc.).  This should remain "the reader."